webpack/react_app/Routes/ForemanREXRouter.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import routes from './ForemanREXRoutes';
+
+const ForemanTasksRouter = () => (
+  <Switch>
```

Comment:
instead of using react-router from scratch, you can now register a plugin's route to foreman core router.
related documentation exists in our storybook - `Client Routing` - or in the PR itself https://github.com/theforeman/foreman/pull/6936

Reply:
Now it's much cleaner, thanks!